packages/editor/src/components/provider/disable-non-page-content-blocks.js

```diff
@@ -4,13 +4,14 @@
 import { useSelect, useDispatch } from '@wordpress/data';
 import { store as blockEditorStore } from '@wordpress/block-editor';
 import { useEffect } from '@wordpress/element';
+import { applyFilters } from '@wordpress/hooks';
 
-const CONTENT_ONLY_BLOCKS = [
-	'core/post-content',
-	'core/post-featured-image',
+const CONTENT_ONLY_BLOCKS = applyFilters( 'editor.postContentBlockTypes', [
```

Comment:
To be honest, I'm not a big fan of this filter. I think ultimately, this is probably better as a property of the block. That said, it's hard to make it a property of the block because it doesn't really make sense in a generic block editor context. It's a WP specific property.

Reply:
Im more than happy to experiment more with other approaches before this gets merged to core. 

I took this route because @noisysocks recommended we go with this approach over having another block support. 

I'm not at all set on approach here :)

Reply:
Yeah I'm hesitant to have this feature escape `@wordpress/editor`. It's very WP Admin specific.